### PR TITLE
basic support for cell decoration

### PR DESCRIPTION
**Pull request by samuil-yanovski**
I've added a sample decorator - makes it easy to show some more content in the cells besides the date (or even replace the date).

### CODE REVIEW

**Comment by edenman** [comment]
Sorry for the delay in reviewing this.  Two big things:

- I don't like exposing `MonthCellDescriptor`, especially since it's mutable.  Can you do this with just the Date? Something like `void decorate(CalendarCellView cellView, Date date);`?
- There's a bunch of reformatting noise in this PR and your new code is all 4-space indents.  We use 2-space indents.  Formatter is here if that's easier than manully fixing all of them.  https://github.com/square/java-code-styles

**Comment by samuil-yanovski** [comment]
Thanks for the feedback. :)
I pushed some fixes earlier today. Hopefully it looks better now.
I'll be able to check if I've missed any formatting issues a little later today.

File: library/src/com/squareup/timessquare/CalendarCellDecorator.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.squareup.timessquare;
+
+import java.util.Date;
+
+/**
```
Comment (edenman) [inline library/src/com/squareup/timessquare/CalendarCellDecorator.java]: Kill this comment

File: sample/src/com/squareup/timessquare/sample/SampleDecorator.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.squareup.timessquare.sample;
+
+import android.text.SpannableString;
+import android.text.Spanned;
+import android.text.style.RelativeSizeSpan;
+import com.squareup.timessquare.CalendarCellDecorator;
+import com.squareup.timessquare.CalendarCellView;
+import java.util.Date;
+
+/**
```
Comment (edenman) [inline sample/src/com/squareup/timessquare/sample/SampleDecorator.java]: kill this comment too

File: sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java [hunk 3]
```diff
@@ -10,11 +10,15 @@
 import android.view.View.OnClickListener;
 import android.widget.Button;
 import android.widget.Toast;
+
```
Comment (edenman) [inline sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java]: nit: no newlines between imports

File: sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java [hunk 4]
```diff
@@ -110,8 +121,7 @@ public void onClick(View v) {
 
         calendar.init(new Date(), nextYear.getTime()) //
             .inMode(SelectionMode.SINGLE) //
-            .withSelectedDate(new Date())
-            .displayOnly();
+            .withSelectedDate(new Date()).displayOnly();
```
Comment (edenman) [inline sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java]: Let's either leave this as-is or change it to:
```
.withSelectedDate(new Date()) //
.displayOnly()
```

**Comment by edenman** [comment]
This is looking good, just a few nits.  Fix those and squash your commits and I'll merge.

**Comment by samuil-yanovski** [comment]
Sorry for the delay. :)
Think I've fixed the issues, let me know if you notice anything else.

**Comment by edenman** [comment]
Thanks @samuil-yanovski!

**Comment by angelpinheiro** [comment]
Great job guys! Do you have any idea about when this feature will be available via gradle?

**Comment by edenman** [comment]
Thanks for the reminder: just pushed 1.6.0 to maven central.  Should be live in a couple hours.

**Comment by angelpinheiro** [comment]
That's great, thank you very much!

**Pull request merged**